pkg/api/validation/validation.go

```diff
@@ -30,12 +30,29 @@ import (
 	"github.com/golang/glog"
 )
 
+const MinPort int = 1
+const MinHostPort int = 0
+const MaxPort int = 65536
+
+const intervalValueErrorMsg string = "must be %d or more and less than %d"
+
+const QualifiedNameErrorMsg string = "must match regex [" + util.Dns1123SubdomainFmt + " / ] " + util.Dns1123LabelFmt
+const DnsSubdomainErrorMsg string = "must match regex " + util.Dns1123SubdomainFmt
+const DnsLabelErrorMsg string = "must match regex " + util.Dns1123LabelFmt
+const PortNameErrorMsg string = "must be less than %d characters and match regex " + util.Dns1123LabelFmt + " pattern"
+const CIdentifierErrorMsg string = "have to match " + util.CIdentifierFmt + " pattern"
+const NegativeErrorMsg string = "value cannot be negative"
+
+var PdPartitionErrorMsg string = fmt.Sprintf(intervalValueErrorMsg, 0, 255)
```

Comment:
it's just too easy to lose track of what % values are in it when it is a constant.